src/elements/ops.rs

```diff
@@ -369,6 +369,158 @@ pub enum Instruction {
 	I64AtomicRmwCmpxchg8u(MemArg),
 	I64AtomicRmwCmpxchg16u(MemArg),
 	I64AtomicRmwCmpxchg32u(MemArg),
+
+	V128Const(Box<[u8; 16]>),
+	V128Load(MemArg),
+	V128Store(MemArg),
+	I8x16Splat,
+	I16x8Splat,
+	I32x4Splat,
+	I64x2Splat,
+	F32x4Splat,
+	F64x2Splat,
+	I8x16ExtractLaneS(u8),
+	I8x16ExtractLaneU(u8),
+	I16x8ExtractLaneS(u8),
+	I16x8ExtractLaneU(u8),
+	I32x4ExtractLane(u8),
+	I64x2ExtractLane(u8),
+	F32x4ExtractLane(u8),
+	F64x2ExtractLane(u8),
+	I8x16ReplaceLane(u8),
+	I16x8ReplaceLane(u8),
+	I32x4ReplaceLane(u8),
+	I64x2ReplaceLane(u8),
+	F32x4ReplaceLane(u8),
+	F64x2ReplaceLane(u8),
+	V8x16Shuffle(Box<[u8; 16]>),
+	I8x16Add,
+	I16x8Add,
+	I32x4Add,
+	I64x2Add,
+	I8x16Sub,
+	I16x8Sub,
+	I32x4Sub,
+	I64x2Sub,
+	I8x16Mul,
+	I16x8Mul,
+	I32x4Mul,
+	// I64x2Mul,
+	I8x16Neg,
+	I16x8Neg,
+	I32x4Neg,
+	I64x2Neg,
+	I8x16AddSaturateS,
+	I8x16AddSaturateU,
+	I16x8AddSaturateS,
+	I16x8AddSaturateU,
+	I8x16SubSaturateS,
+	I8x16SubSaturateU,
+	I16x8SubSaturateS,
+	I16x8SubSaturateU,
+	I8x16Shl,
+	I16x8Shl,
+	I32x4Shl,
+	I64x2Shl,
+	I8x16ShrS,
+	I8x16ShrU,
+	I16x8ShrS,
+	I16x8ShrU,
+	I32x4ShrS,
+	I32x4ShrU,
+	I64x2ShrS,
+	I64x2ShrU,
+	V128And,
+	V128Or,
+	V128Xor,
+	V128Not,
+	V128Bitselect,
+	I8x16AnyTrue,
+	I16x8AnyTrue,
+	I32x4AnyTrue,
+	I64x2AnyTrue,
+	I8x16AllTrue,
+	I16x8AllTrue,
+	I32x4AllTrue,
+	I64x2AllTrue,
+	I8x16Eq,
+	I16x8Eq,
+	I32x4Eq,
+	// I64x2Eq,
+	F32x4Eq,
+	F64x2Eq,
+	I8x16Ne,
+	I16x8Ne,
+	I32x4Ne,
+	// I64x2Ne,
+	F32x4Ne,
+	F64x2Ne,
+	I8x16LtS,
+	I8x16LtU,
+	I16x8LtS,
+	I16x8LtU,
+	I32x4LtS,
+	I32x4LtU,
+	// I64x2LtS,
+	// I64x2LtU,
+	F32x4Lt,
+	F64x2Lt,
+	I8x16LeS,
+	I8x16LeU,
+	I16x8LeS,
+	I16x8LeU,
+	I32x4LeS,
+	I32x4LeU,
+	// I64x2LeS,
+	// I64x2LeU,
+	F32x4Le,
+	F64x2Le,
+	I8x16GtS,
+	I8x16GtU,
+	I16x8GtS,
+	I16x8GtU,
+	I32x4GtS,
+	I32x4GtU,
+	// I64x2GtS,
+	// I64x2GtU,
+	F32x4Gt,
+	F64x2Gt,
+	I8x16GeS,
+	I8x16GeU,
+	I16x8GeS,
+	I16x8GeU,
+	I32x4GeS,
+	I32x4GeU,
+	// I64x2GeS,
```

Comment:
commented out? why?

Reply:
Ah I'm fine either way on this, these are basically "holes" in the spec where the instruction isn't specified (but this is the instruction it would be otherwise). I believe this is done to be conservative in the first pass of SIMD and the instructions may be added later. I don't mind removing them though if it's confusing!